docs/MAINTAINERS.adoc

```diff
@@ -161,6 +167,157 @@ Code reviews do NOT pursue the following goals:
+  Such ratings allow users to reference issues they experienced with new Jenkins core releases,
+  and it helps to discover regressions in the core causing instability or unexpected plugin failures.
+* link:https://issues.jenkins-ci.org/secure/Dashboard.jspa?selectPageId=20742[Jenkins core triage board] -
+  Lists untriaged and recent issues in the Jenkins core and bundled components.
+* link:https://issues.jenkins-ci.org/secure/Dashboard.jspa?selectPageId=20340[Core maintainers board] -
+  Lists unresolved recent regressions, unresolved recent core bugs, and popular new issues.
+  This dashboard can be used to discover issues that **might** be related to the recent changes in the Jenkins core.
+
+=== HOWTO: Initial triage
+
+Initial issue triage follows the following objectives:
+
+* **Perform initial triage of an issue**.
+  Issue triage team members are not expected to perform a full analysis of the issue (though they are welcome to do so!).
+  The main goal is to ensure that an issue report is legitimate and that it contains enough information to be processed.
+  It is fine to request additional information from submitters and/or to refer them to reporting guidelines:
+** link:https://www.jenkins.io/participate/report-issue/[Guide: How to report an issue in Jenkins]
+** link:https://github.com/jenkinsci/remoting#reporting-issues[Reporting Jenkins Remoting issues]
+* **Verify the issue component**.
+  It is essential to ensure that the `component` field references the right component (the Jenkins core, a plugin, etc.)
+  so that an issue is discoverable processed by a component maintainer.
+  When moving an issue, assign the issue to the `automatic` assignee so that the maintainer gets notification.
+  Not all components have a default assignee, and it is perfectly fine to leave the assignee empty.
+* **Verify the issue type**.
+  `Bug` should be used for bug reports.
+  All other issue types are considered as requests for enhancements, and there is no practical difference for the Jenkins core.
+* **Verify the issue metadata**: Jenkins version, environment, labels, etc.
+  Such metadata is useful for further triage and issue discoverability.
+  There are some labels used in Jenkins Jira dashboard and filters, e.g. `jcasc-compatibility`, `java11-compatibility`, `jep-200`, etc.
+  Assigning such labels helps users and maintainers to discover issues and act on them.
+  There is no list of such "common labels" recommended for use.
+  Some labels can be found in similar issues or documentation linked from system log entries in the reports.
+* **Move security issue** to the `SECURITY` project.
+  Sometimes the issue reporters do not follow the link:https://www.jenkins.io/security/#reporting-vulnerabilities[vulnerability reporting] process and report security issues in public.
+  If you see such issues, move them to the `SECURITY` project so that the security team takes care of their triage.
```

Comment:
Do all of the fields get migrated correctly between JENKINS and SECURITY? Sometimes when moving between projects, Jira loses some information.

Reply:
```suggestion
  If you see such issues, move them to the `SECURITY` project so that the security team takes care of their triage.
  Note that the required fields are different between projects, so some manual updates might be required when moving them.
```

WDYT @slide?

Reply:
LGTM